src/common/controls/Control.js

```diff
@@ -14,6 +14,7 @@ class Control extends Observable {
   constructor(map, options = {}) {
     super();
     this.map = map;
+    this.targetElement = options.targetElement || this.map.getTargetElement();
```

Comment:
Do you agree on setting the map target element as default target for the copyright?